### PR TITLE
Refactor test files

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
This is to rename the ballerina files of the test cases in order to identify the use case just by looking at the file name and add new use cases without conflicting.

### CODE REVIEW

**Comment by chrishantha** [comment]
`h1` and `h2` names are not clear. File name length s not a problem. Could you please use more meaningful names?

**Comment by ldclakmal** [comment]
@chrishantha These names were used for following abbrevations (from the spec https://tools.ietf.org/html/rfc7540) in order to add new use cases without conflicting the file names.
- h2 - HTTP/2 over TLS
- h2c - HTTP/2 over cleartext
- h1 - HTTP/1.1 over TLS
- h1c - HTTP/1.1 over cleartext

If they are not clear, the longer names would be as follows:
- http1_http1_passthorough.bal
- http1_cleartext_http1_cleartext_passthorough.bal
- http2_http1_passthorough.bal
- http2_http2_passthorough.bal

WDYT?

**Comment by VijithaEkanayake** [comment]
@ldclakmal I prefer to use http1x and http2 rather than using h1 and h2. Also, I think we don't need to repeat the abbreviation in the file if there is no protocol switch. We can just say http2_passthorough.bal instead of http2_http2_passthorough.bal

WDYT?

**Comment by ldclakmal** [comment]
@VijithaEkanayake Then the following _Set 2_ will be the updated set according to the suggested format. Is that okay?

Set 1 | Set 2
-- | --
h1_h1 | http1s_passthrough
h1c_h1c | http1_passthrough
h2_h1 | http2s_http1s_passthrough
h2_h1c | http2s_http1_passthrough
h2_h2 | http2s_passthrough

**Comment by arukshani** [comment]
Personally I like the shorter names since it is easy to get the meaning at a glance and h2, h2c these come from the spec itself.  These abbreveations are in use in the industry if you look at [1]

[1] https://line.github.io/armeria/apidocs/com/linecorp/armeria/common/SessionProtocol.html

But since it seems that majority are not familiar with the spec and these abbreveations let's use the set 2 for the names.

**Comment by shafreenAnfar** [comment]
Even though I am usually against enigmatic names, I believe in this case we must go with short names as in the spec. I know it sounds foreign to us but for people with HTTP domain literacy it sounds normal.

**Comment by VijithaEkanayake** [comment]
@ldclakmal Seems these shorter names are widely used in the HTTP domain even in the spec too. Hence +1 to go with the shorter names in this case.

File: distribution/scripts/jmeter/run-performance-tests.sh [hunk 1]
```diff
@@ -33,10 +33,10 @@ function initialize() {
 export -f initialize
 
 declare -A test_scenario0=(
-    [name]="passthrough_http"
-    [display_name]="Passthrough HTTP service"
-    [description]="An HTTP Service, which forwards all requests to a back-end service."
-    [bal]="passthrough.balx"
+    [name]="h1c_h1c_passthrough"
+    [display_name]="Passthrough HTTP service (h1c -> h1c)"
+    [description]="A HTTP Service, which forwards all requests to a HTTP back-end service."
```
Comment (chrishantha) [inline distribution/scripts/jmeter/run-performance-tests.sh]: "An HTTP" should be correct. Search in https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol. There are other references as well: https://www.google.com/search?q=%22an+http%22

Comment (ldclakmal) [reply distribution/scripts/jmeter/run-performance-tests.sh]: Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/445f1a74637b7636869110acc99d20af08a49923

File: distribution/scripts/jmeter/run-performance-tests.sh [hunk 2]
```diff
@@ -83,10 +83,10 @@ declare -A test_scenario3=(
     [skip]=false
 )
 declare -A test_scenario4=(
-    [name]="passthrough_http2_https"
-    [display_name]="Passthrough HTTP2 (HTTPS) service"
-    [bal]="http2_https_passthrough.balx"
-    [description]="An HTTPS Service exposed over HTTP2 protocol, which forwards all requests to a HTTP2 (HTTPS) back-end service."
+    [name]="h2_h2_passthrough"
+    [display_name]="Passthrough HTTP2 (HTTPS) service (h2 -> h2)"
```
Comment (arukshani) [inline distribution/scripts/jmeter/run-performance-tests.sh]: Feels like this display name is misleading. For HTTP/1.1 over TLS scenario its called HTTPS  and here also in brackets, it says HTTPS which gives a wrong meaning. I would rather use "over TLS" inside brackets.

Comment (ldclakmal) [reply distribution/scripts/jmeter/run-performance-tests.sh]: Agreed. Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/0b4d53cff493f87629996209363f02f91919c66d

File: distribution/scripts/jmeter/run-performance-tests.sh [hunk 3]
```diff
@@ -70,7 +70,7 @@ declare -A test_scenario2=(
     [skip]=false
 )
 declare -A test_scenario3=(
-    [name]="transformation_https"
+    [name]="https_transformation"
```
Comment (chrishantha) [inline distribution/scripts/jmeter/run-performance-tests.sh]: Should `https_transformation` be renamed as `h1_transformation`? Similarly `http_transformation` as `h1c_transformation`?

Comment (ldclakmal) [reply distribution/scripts/jmeter/run-performance-tests.sh]: Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/7348d0132de911748e34cb165d0c0ac5a539f399

**Pull request merged**